build/yamls/antrea-aks.yml

```diff
@@ -174,6 +174,169 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
+metadata:
+  labels:
+    app: antrea
+  name: externalentities.core.antrea.tanzu.vmware.com
+spec:
+  group: core.antrea.tanzu.vmware.com
+  names:
+    kind: ExternalEntity
```

Comment:
I am not suggesting a change now. Just want to mention @abhiraut has a good point that later we might have both Cluster scope and Namespace scope ExternalEntity/ExternalNode, then naming will become a problem again.
If you guys have any good ideas, we can change the names earlier too. @abhiraut @tnqn

Reply:
yeah. lets do that before this release at least. will propose something

Reply:
curious,  why would cluster scope ExternalEntity be a problem later? can we just name them ExternalEntity and ClusterExternalEntity respectively ?